src/shader.cpp

```diff
@@ -826,6 +829,9 @@ ShaderInfo generate_shader(std::string name, u8 material_type, u8 drawtype,
 					"failed to generate \""<<name<<"\", "
 					"addShaderMaterial failed."
 					<<std::endl;
+			dumpShaderProgram("Vertex", vertex_program);
+			dumpShaderProgram("Pixel", pixel_program);
+			dumpShaderProgram("Geometry", geometry_program);
```

Comment:
The geometry shader isn't passed to addShaderMaterial, so it shouldn't be printed here.